scene/realistic-lighting-and-shadows/src/main/resources/realistic_lighting_and_shadows/main.fxml

```diff
@@ -33,9 +36,15 @@
     <!--Slider box-->
     <VBox styleClass="panel-region">
         <Label fx:id="time"/>
-        <Button fx:id="noSunButton" text="No sun light effect" maxWidth="Infinity"/>
-        <Button fx:id="sunOnlyButton" text="Sun light only" maxWidth="Infinity"/>
-        <Button fx:id="sunAndShadowsButton" text="Sun light with shadows" maxWidth="Infinity"/>
+        <ComboBox fx:id="comboBox" maxWidth="Infinity">
+            <items>
+                <FXCollections fx:factory="observableArrayList">
+                    <LightingMode fx:value="NO_LIGHT"/>
```

Comment:
Great find on this! I didn't know it's possible to populate runtime API values into this using FXML 👍